### PR TITLE
Purge and tenantParameters for old-style disable/enable calls OKAPI-831

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

File: okapi-core/src/main/raml/okapi.raml [hunk 1]
```diff
@@ -706,6 +715,15 @@ types:
...
+          queryParameters:
+            purge:
+              description: Disabled modules will also be purged.
+              type: boolean
+              required: false
+            tenantParameters:
+              description: Parameters for Tenant init
+              type: string
+              required: false
           body:
```
Comment (hjiebsco) [inline okapi-core/src/main/raml/okapi.raml:727]: Does this section (mdTo will not be empty, right?) conflict with line 400 `boolean purge = mdTo == null && options.getPurge();`

File: okapi-core/src/main/raml/okapi.raml [hunk 2]
```diff
@@ -666,6 +666,15 @@ types:
...
         description: Disable modules for tenant
+        queryParameters:
+          purge:
+            description: Disabled modules will also be purged.
+            type: boolean
+            required: false
+          tenantParameters:
+            description: Parameters for Tenant init
+            type: string
+            required: false
```
Comment (hjiebsco) [inline okapi-core/src/main/raml/okapi.raml:677]: Maybe add a purge test for this one too?

File: okapi-core/src/main/raml/okapi.raml (hunk 1 above)
Comment (adamdickmeiss) [reply okapi-core/src/main/raml/okapi.raml:727]: purge would always be false in this case. mdTo == null ONLY if a module is disabled (no downgrade or upgrade). The purge would be ignored essentially.

File: okapi-core/src/main/raml/okapi.raml (hunk 2 above)
Comment (adamdickmeiss) [reply okapi-core/src/main/raml/okapi.raml:677]: Could do.. but that would mostly be to check that RAML is ok.. There's 100% coverage.

File: okapi-core/src/main/raml/okapi.raml (hunk 1 above)
Comment (hjiebsco) [reply okapi-core/src/main/raml/okapi.raml:727]: So this part should be modified because essentially this API does not support "purge" parameter, right?

**Pull request merged**